### ISSUE TITLE
getAllTransactions should return Array<Account>

### ISSUE THREAD

**Issue by rjames86** [plaid/plaid-node#163]
Currently the response type for `getAllTransactions` is an array of transactions. This requires making an additional call to get the list of accounts associated with the transactions. Proposing possibly a new response type for all transactions like

```
  // omitting extending the BaseResponse since there isn't a single request_id
  interface TransactionsAllResponse {
    accounts: Array<Account>;
    total_transactions: number;
    transactions: Array<Transaction>;
    item: Item;
  }

```

**Comment by freewil** [comment]
For simplicity and flexibility, I've proposed just having `getAlTransactions` return an array of responses. This allows the maximum amount of flexibility. See my branch:

https://github.com/plaid/plaid-node/compare/master...hellodigit:get-all-transactions-responses

**Comment by ntindall** [comment]
Fixed with https://github.com/plaid/plaid-node/releases/tag/2.7.0

**Comment by andrewheekin** [comment]
Any progress on this one? Would love to see getAllTransactions return the same TransactionsResponse as getTransactions:
```
  interface TransactionsResponse extends BaseResponse {
    accounts: Array<Account>;
    total_transactions: number;
    transactions: Array<Transaction>;
    item: Item;
  }
```

**Comment by andrewheekin** [comment]
It looks like pr #183 addresses this. Will this be merged?

**Comment by ntindall** [comment]
I think we can include this in the next breaking change to the client. Thanks for flagging!

**Comment by andrewheekin** [comment]
@ntindall great!

**Comment by ntindall** [comment]
Resolved with https://github.com/plaid/plaid-node/pull/203 and published with 4.0.0
